cw/capture.py

```diff
@@ -1039,80 +1102,85 @@ def capture_ecdsa_simple(ot, fw_bin, pll_frequency, capture_cfg):
     # Loop to collect each power trace
     for _ in tqdm(range(capture_cfg["num_traces"]), desc='Capturing',
                   ncols=80):
-        # create a clean array to keep each section
+        
```

Comment:
This part sets the inputs. This template can be used for the new ECDSA commands.